### DISCUSSION TITLE
How do I keep the whole treetable expanded?

### DISCUSSION THREAD

**Discussion by yashwp** [primefaces/community#2529] [PrimeVue]
Is there a way in the configuration to keep the TreeTable expanded as default?

Expected Behaviour - 

<img width="658" alt="image" src="https://github.com/user-attachments/assets/32f364d5-c0a5-4728-a056-7cd09add967d">

**Comment by ShiYaoDiving** [comment]
You can use the v-model:expanded-keys and set datakey all true.

**Reply by yashwp** [reply]
I've a dynamic object. I need to unnecessarily loop over the object and set those keys to true. There should be an option in the configuration itself.
